implementations/rust/ockam/ockam_transport_udp/src/router/messages.rs

```diff
@@ -0,0 +1,13 @@
+use ockam_core::{Address, Message};
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Debug, Clone, Hash, Ord, PartialOrd, Eq, PartialEq, Message)]
+pub(crate) enum UdpRouterMessage {
+    /// Register a new client to this routing scope.
+    Register {
+        /// Specify an accept scope for this client.
+        accepts: Vec<Address>,
+        /// The clients own worker bus address.
+        self_addr: Address,
+    },
+}
```

Comment:
This is definitely out of scope for this PR but we should think about having _one_ of these types in `ockam_core` instead of one for each transport

Reply:
Sure, I am willing to open a followup PR for this